src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/ConnectionHandlerContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.IO;
+using Microsoft.Data.SqlClient.SNI;
+using Microsoft.Data.SqlClientX.Handlers;
+
+namespace Microsoft.Data.SqlClient.Microsoft.Data.SqlClientX.Handlers
```

Comment:
```suggestion
namespace Microsoft.Data.SqlClientX.Handlers
```

VS has generated lot of bad such namespaces for us :)

Reply:
I can fix that :) We basically just need to remove the default namespace from the project files. I'll send that out for a PR against main

Reply:
Awesome! thanks Ben. I'll update the file for now.